src/views/admin/AdminView.tsx

```diff
@@ -23,20 +24,63 @@ export const AdminView = () => {
       window.location.reload()
     }
   }
+
+  const handleExportClick = () => {
+    const fileName = 'atlas_local_state'
+    const state = Object.keys(localStorage).reduce((obj, k) => {
+      return { ...obj, [k]: localStorage.getItem(k) }
+    }, {})
+
+    const json = JSON.stringify(state)
+    const blob = new Blob([json], { type: 'application/json' })
+    const href = URL.createObjectURL(blob)
+    const link = document.createElement('a')
+    link.href = href
+    link.download = fileName + '.json'
+    document.body.appendChild(link)
+    link.click()
+    document.body.removeChild(link)
```

Comment:
This won't fully work because you're treating read items as just regular text and those are already JSON objects. Values will be raw strings in that JSON file and not proper objects. Probably doing `JSON.parse` on `locaStorage.getItem()` result may do the trick

Reply:
Take a look at scripts I've prepared here: https://gist.github.com/kdembler/a77b32bc1ee7b8898e3d07cf34e26d26#file-atlas-dev-test-md

Reply:
Did this correctly at the beginning but for some reason it didn't work. Same with the scripts that you've provided - if there is `target_env` present in the localStorage the scripts will fail. Added fix for storing env, so should be good now